RecoMuon/TrackerSeedGenerator/plugins/TSGForOIFromL2.cc

```diff
@@ -0,0 +1,474 @@
+ */
+
+#include "RecoMuon/TrackerSeedGenerator/plugins/TSGForOIFromL2.h"
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "DataFormats/Math/interface/deltaR.h"
+
+#include <memory>
+
+using namespace edm;
+using namespace std;
+
+TSGForOIFromL2::TSGForOIFromL2(const edm::ParameterSet & iConfig) :
+  src_(consumes<reco::TrackCollection>(iConfig.getParameter<edm::InputTag>("src"))),
+  maxSeeds_(iConfig.getParameter<uint32_t>("maxSeeds")),
+  maxHitlessSeeds_(iConfig.getParameter<uint32_t>("maxHitlessSeeds")),
+  maxHitSeeds_(iConfig.getParameter<uint32_t>("maxHitSeeds")),
+  numOfLayersToTry_(iConfig.getParameter<int32_t>("layersToTry")),
+  numOfHitsToTry_(iConfig.getParameter<int32_t>("hitsToTry")),
+  numL2ValidHitsCutAllEta_(iConfig.getParameter<uint32_t>("numL2ValidHitsCutAllEta")),
+  numL2ValidHitsCutAllEndcap_(iConfig.getParameter<uint32_t>("numL2ValidHitsCutAllEndcap")),  
+  fixedErrorRescalingForHits_(iConfig.getParameter<double>("fixedErrorRescaleFactorForHits")),
+  fixedErrorRescalingForHitless_(iConfig.getParameter<double>("fixedErrorRescaleFactorForHitless")),
+  adjustErrorsDynamicallyForHits_(iConfig.getParameter<bool>("adjustErrorsDynamicallyForHits")),
+  adjustErrorsDynamicallyForHitless_(iConfig.getParameter<bool>("adjustErrorsDynamicallyForHitless")),
+  estimatorName_(iConfig.getParameter<std::string>("estimator")),
+  minEtaForTEC_(iConfig.getParameter<double>("minEtaForTEC")),
+  maxEtaForTOB_(iConfig.getParameter<double>("maxEtaForTOB")),
+  useHitLessSeeds_(iConfig.getParameter<bool>("UseHitLessSeeds")),
+  updator_(new KFUpdator()), measurementTrackerTag_(consumes<MeasurementTrackerEvent>(iConfig.getParameter<edm::InputTag>("MeasurementTrackerEvent"))),
+  pT1_(iConfig.getParameter<double>("pT1")),
+  pT2_(iConfig.getParameter<double>("pT2")),
+  pT3_(iConfig.getParameter<double>("pT3")),
+  eta1_(iConfig.getParameter<double>("eta1")),
+  eta2_(iConfig.getParameter<double>("eta2")),
+  eta3_(iConfig.getParameter<double>("eta3")),
+  eta4_(iConfig.getParameter<double>("eta4")),
+  eta5_(iConfig.getParameter<double>("eta5")),
+  eta6_(iConfig.getParameter<double>("eta6")),
+  eta7_(iConfig.getParameter<double>("eta7")),
+  SF1_(iConfig.getParameter<double>("SF1")),
+  SF2_(iConfig.getParameter<double>("SF2")),
+  SF3_(iConfig.getParameter<double>("SF3")),
+  SF4_(iConfig.getParameter<double>("SF4")),
+  SF5_(iConfig.getParameter<double>("SF5")),
+  SF6_(iConfig.getParameter<double>("SF6")),
+  tsosDiff1_(iConfig.getParameter<double>("tsosDiff1")),
+  tsosDiff2_(iConfig.getParameter<double>("tsosDiff2")),
+  propagatorName_(iConfig.getParameter<std::string>("propagatorName")), 
+  theCategory(string("Muon|RecoMuon|TSGForOIFromL2"))
+{
+  produces<std::vector<TrajectorySeed> >();
+}
+
+
+TSGForOIFromL2::~TSGForOIFromL2() {
+
+}
+
+
+//
+// Produce seeds
+//
+void TSGForOIFromL2::produce(edm::StreamID sid, edm::Event& iEvent, const edm::EventSetup& iSetup) const {
+    
+  using namespace std;
```

Comment:
This duplicates the using declaration in the outer scope (which is probably not needed anyhow)
Please remove, and also remove a few empty lines for an easier reading of the code